$MirOS: ports/www/dillo/stable/patches/patch-dpi_bookmarks_c,v 1.2 2005/06/19 17:20:19 bsiegert Exp $
--- dpi/bookmarks.c.orig	Sat Jan 21 01:26:59 2006
+++ dpi/bookmarks.c	Mon May 15 17:49:33 2006
@@ -728,7 +728,7 @@ static void Bms_check_import(void)
 
    if (access(BmFile, F_OK) != 0) {
       OldBmFile = g_strconcat(g_get_home_dir(),
-                          "/", ".dillo/bookmarks.html", NULL);
+                          "/", ".etc/dillo/bookmarks.html", NULL);
       if (access(OldBmFile, F_OK) == 0) {
          g_string_sprintf(gstr, cmd1, BmFile);
          system(gstr->str);
@@ -1738,7 +1738,7 @@ int main (void) {
    if (signal (SIGTERM, termination_handler) == SIG_IGN)
      signal (SIGTERM, SIG_IGN);
 
-   BmFile = g_strconcat(g_get_home_dir(), "/", ".dillo/bm.txt", NULL);
+   BmFile = g_strconcat(g_get_home_dir(), "/", ".etc/dillo/bm.txt", NULL);
 
    g_print("bookmarks.dpi (v.13): accepting connections...\n");
 
